### PR TITLE
Microsoft.TeamMate version 0.1.4.0

### PR DESCRIPTION
**Pull request by supermem613**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55310)

### CODE REVIEW

File: manifests/m/Microsoft/TeamMate/0.1.4.0/Microsoft.TeamMate.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- ado
+- azuredevops
+- teammate
+ReleaseNotes: >
+  * Fix all tile text to use the font color by @supermem613 in https://github.com/microsoft/TeamMate/pull/67
+
+  * Update nuget dependencies to latest versions by @supermem613 in https://github.com/microsoft/TeamMate/pull/69
+
+  * Extend PR queries to allow 'Created by' and 'Assigned to' with '@me' by @supermem613 in https://github.com/microsoft/TeamMate/pull/71
+ReleaseNotesUrl: https://github.com/microsoft/TeamMate/releases/tag/0.1.2-ci.4%2BBranch.main.Sha.d7c0ecf5d886326ecd14eab6c9b6283000aaef1d
```
Comment (Trenly) [inline manifests/m/Microsoft/TeamMate/0.1.4.0/Microsoft.TeamMate.locale.en-US.yaml:29]:
```suggestion
ReleaseNotesUrl: https://github.com/microsoft/TeamMate/releases/tag/0.1.4-ci.2%2BBranch.main.Sha.9aea9d582ead1955134442dca1fc68d47ac9b41e
```

File: manifests/m/Microsoft/TeamMate/0.1.4.0/Microsoft.TeamMate.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Tags:
+- ado
+- azuredevops
+- teammate
+ReleaseNotes: >
+  * Fix all tile text to use the font color by @supermem613 in https://github.com/microsoft/TeamMate/pull/67
+
+  * Update nuget dependencies to latest versions by @supermem613 in https://github.com/microsoft/TeamMate/pull/69
+
+  * Extend PR queries to allow 'Created by' and 'Assigned to' with '@me' by @supermem613 in https://github.com/microsoft/TeamMate/pull/71
```
Comment (Trenly) [inline manifests/m/Microsoft/TeamMate/0.1.4.0/Microsoft.TeamMate.locale.en-US.yaml:28]:
```suggestion
ReleaseNotes: >
  * Improve search experience by @supermem613 in https://github.com/microsoft/TeamMate/pull/80
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by supermem613** [comment]
@Trenly, thank you!

**Review by OfficialEsco — APPROVED** [review]

**Comment by OfficialEsco** [comment]
GitHub is just having a stroke https://www.githubstatus.com

**Pull request merged**